From c0caa5c97a0ffe73125351cfc3008ffa0e8d1ea0 Mon Sep 17 00:00:00 2001
From: Don Olmstead <don.olmstead@sony.com>
Date: Tue, 13 Oct 2020 13:50:17 -0700
Subject: [PATCH] Adjust CMake for vcpkg

Changes the locations of installed files to match what vcpkg expects.
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cd04b9745..8cbc69a88 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -623,12 +623,12 @@ if (NOT SKIP_INSTALL_LIBRARIES AND NOT SKIP_INSTALL_ALL)
       COMPONENT libraries)
   install(
     EXPORT freetype-targets
-      DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/freetype
+      DESTINATION ${CMAKE_INSTALL_LIBDIR}/freetype/cmake
       FILE freetype-config.cmake
       COMPONENT headers)
   install(
     FILES ${PROJECT_BINARY_DIR}/freetype-config-version.cmake
-    DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/freetype
+    DESTINATION ${CMAKE_INSTALL_LIBDIR}/freetype/cmake
     COMPONENT headers)
 endif ()
 
-- 
2.36.0.windows.1

